src/blocks/navigation.scss

```diff
@@ -0,0 +1,118 @@
+body {
+  margin: 0;
+}
+
+.logo__image {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+:root {
+  --color-hover: #00acdc;
+  --color-default: #000;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 12px;
+  font-family: Roboto, serif;
+  font-weight: 500;
+  padding: 0 50px;
+  padding-inline: 50px;
```

Comment:
The padding property is defined twice for the .header class. You can remove one of them to avoid redundancy. Consider keeping only `padding-inline: 50px;` if you want to apply padding to the left and right sides.